### ISSUE TITLE
Documentation published somewhere?

### ISSUE THREAD

**Issue by hug-dev** [parallaxsecond/parsec-client-go#7]
Would it be possible to publish the Go client somewhere so that its documentation is visible? Something like [that](https://pkg.go.dev/crypto)?

**Comment by hug-dev** [comment]
Also, let's aim at having documentation on all exported items 🤙

**Comment by jn9e9** [comment]
Agreed, have linter complaining madly about not having that level of docs.  Can add docs following instructions here: https://go.dev/about#adding-a-package
